src/cloud/cdn/configure-fastly.md

```diff
@@ -9,18 +9,24 @@ functional_areas:
   - Configuration
 ---
 
-[Fastly]({{ site.baseurl }}/cloud/cdn/cloud-fastly.html) is required for {{site.data.var.ece}}, and is used in Staging and Production environments. It works with Varnish to provide fast caching capabilities and a [Content Delivery Network](https://glossary.magento.com/content-delivery-network) (CDN) for static assets. Fastly is not available in Integration environments.
+[Fastly]({{ site.baseurl }}/cloud/cdn/cloud-fastly.html) is required for {{site.data.var.ece}}, and is used in Staging and Production environments. It works with Varnish to provide fast caching capabilities and a [Content Delivery Network](https://glossary.magento.com/content-delivery-network) (CDN) for static assets. Fastly also provides a Web Application Firewall (WAF) to secure your site and Cloud infrastructure. You must route all incoming site traffic through Fastly to protect your site and Cloud infrastructure from malicious traffic and attacks.
 
-This information gets you started with enabling and configuring Fastly caching services in your Staging and Production environments. We provide additional information for configuring backends and Origin shields, customizing error pages, and adding custom VCL snippets.
+{:.bs-callout-info}
+Fastly is not available in Integration environments.
 
-For VCL snippets, experience developing that code is required for advanced configurations.
-
-The process for configuring Fastly includes:
+Complete the following steps to enable, configure, and test Fastly early in your site development process to enable secure access to your site.
 
 -  Get Fastly credentials for Staging and Production environments
 -  Enable Fastly CDN caching in your environment
 -  Upload Fastly VCL snippets
--  Advanced configurations including VCL snippets, as needed for your {{ site.data.var.ee }} sites
+-  Test Fastly caching in pre-production environments
```

Comment:
If it is not available in Integration environments, then maybe this should say "in Staging environment" ?? Just wondering. It probably doesn't matter, because maybe people don't think of Integration as a pre-prod... 🤔

Reply:
In forthcoming instructions that will be linked, the pre-production environment URLs are defined, so I think this is OK to leave as is.